test/user.test.js

```diff
@@ -214,6 +214,24 @@ describe('User', function() {
       assert(u2.password === u1.password);
     });
 
+    it('invalidates the user\'s accessToken when the user is deleted', function(done) {
+      User.create({ email: 'b@c.com', password: 'bar' }, function(err, instance) {
+        User.login({ email: 'b@c.com', password: 'bar' }, function(err, accessToken) {
+          if (err) return done(err);
+          assert(accessToken.userId);
+          User.deleteById(instance.id, function(err) {
+            if (err) return done(err);
+            AccessToken.findById(accessToken.userId, function(err, notFound) {
```

Comment:
Also `AccessToken.findById(userId)` does not make sense to me - it says "find me an access token that has the same id as this user". You should find all tokens belonging to the user instead.

```js
AccessToken.find({ where: { userId: user.id } }, function(err, tokens) {
  // assert no tokens were found
});
```